Tools/BlockTypePaletteGenerator/Generator.lua

```diff
@@ -0,0 +1,83 @@
+-- lib/lunajson/src/ is not in default Lua package paths
+package.path = 'lib/lunajson/src/?.lua;' .. package.path;
+
+function usage(message)
+	if message then
+		io.output(io.stderr);
+		io.write(message, "\n\n");
+	end
+	io.write("Generator.lua - Converts Minecraft report format to cuberite bl"..
+		"ock type palette format.\n"..
+		"\n"..
+		"Usage: lua Generator.lua <inputfile> <outputfile>\n"..
+		"\n"..
+		"<inputfile> and <outputfile> must point to a valid path. <inputfile>"..
+		" must be readable and outputfile must be writable. Either can be rep"..
+		"laced with `-` (dash character) to point to standard input or output"..
+		".");
```

Comment:
You have your dromedaryCase, and I like block justify ;)

``[[ ]]`` would either inject tabs or require the text to stick on the left, because it doesn't work like python's. Also it would introduce any line breaks into the output.